scripts/test-with-coverage.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -e
+
+CWD="$(cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")" && pwd -P)"
+
+cd $CWD/../
+
+PACKAGE="$1"
+echo "$PACKAGE"
+
+source scripts/start_dev_db.sh
+
+# tests are executed in working directory crates/api (or similar),
+# so to load the config we need to traverse to the repo root
+export LEMMY_CONFIG_LOCATION=../../config/config.hjson
+export RUST_BACKTRACE=1
+
+cargo install cargo-llvm-cov
+cargo llvm-cov --workspace --all-features --no-fail-fast --lcov --output-path lcov.info
```

Comment:
Im not sure how to open the lcov file. Why not use `--html --open`?

Reply:
Lcov file can be used to show coverage in the code editor (e.g. with coverage gutters extension for vscode)

Reply:
Would be good if you explain that in a comment. I tried the gutters extension but it fails to find the coverage file, even after I manually configure it to use lcov.info. Can you share your plugin config?

Reply:
I'm using the default config.

Did you click "Watch" at the bottom and open a Rust file? If it says "No Coverage" then that's just referring to the current file.

Reply:
Added comment

Reply:
After clicking "watch" there is an error message "Could not find a Coverage file! Searched for lcov.info, cov.xml, coverage.xml, jacoco.xml, coverage.cobertura.xml". But lcov.info definitely exists and the open file has various tests, so no idea what the problem is. Im using VSCodium in case that makes a difference.